_data/menu_v2.yml

```
@@ -19,6 +19,7 @@ Usage:
 Adapters:
     Local: '/v2/docs/adapter/local/'
     AWS S3: '/v2/docs/adapter/aws-s3-v3/'
+    AsyncAws S3: '/v2/docs/adapter/async-aws-s3/'
```

I know the class is named `AsyncAws`, but the flysystem adapter for S3 is named `AWS`, so I propose to rename the adapter name to:

```suggestion
    AsyncAWS S3: '/v2/docs/adapter/async-aws-s3/'
```

---

It should be "AsyncAws" or "async-aws" depending on the context. That is something we decided just because we think it looks better. 

I know it differs from "AWS".